### PR TITLE
v0.0.10 ❄️

### PR DESCRIPTION
**Pull request by lorepirri**
Upstream: https://github.com/Nixtla/utilsforecast/tree/v0.0.10

https://anaconda.atlassian.net/browse/PKG-3650

Channel: Snowflake

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -37,8 +38,9 @@ about:
   summary: Forecasting utilities
   license: Apache-2.0
   license_file: LICENSE
+  license_family: APACHE
+  description: Forecasting utilities
+  dev_url: https://github.com/Nixtla/utilsforecast
+  doc_url: https://nixtlaverse.nixtla.io/utilsforecast/index.html
+
 
-extra:
```
Comment (M-Waszkiewicz-Anaconda) [inline recipe/meta.yaml:41]: I am not sure why is taken out ?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,27 +1,28 @@
...
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/utilsforecast-{{ version }}.tar.gz
-  sha256: 3a67430243370de78dcb219a4d22000d2e5957c23d5d5f4193d865e7ef3d422a
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 08c01d57a7d221bb0dc34ed72279b6a97795de441adfbe11f4a84ad2ce635c72
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
   number: 0
+  skip: true  # [py<36]
```
Comment (Jrice1317) [inline recipe/meta.yaml]:
```suggestion
  skip: true  # [py<38]
```
https://github.com/Nixtla/utilsforecast/blob/v0.0.10/setup.py#L51
https://github.com/Nixtla/utilsforecast/blob/v0.0.10/settings.ini#L5

Comment (JeanChristopheMorinPerso) [reply recipe/meta.yaml]: Good catch @Jrice1317 !

File: recipe/meta.yaml [hunk 3]
```diff
@@ -37,8 +38,9 @@ about:
   summary: Forecasting utilities
   license: Apache-2.0
   license_file: LICENSE
+  license_family: APACHE
+  description: Forecasting utilities
+  dev_url: https://github.com/Nixtla/utilsforecast
+  doc_url: https://nixtlaverse.nixtla.io/utilsforecast/index.html
+
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml]:
```suggestion
extra:
  recipe-maintainers:
    - AzulGarza
    - jmoralez
```

**Review by JeanChristopheMorinPerso — APPROVED** [review]

**Review by Jrice1317 — APPROVED** [review]

**Pull request merged**